### PR TITLE
Mdip/feature/46 use GitHub actions to automate docker images pushes Closes #46

### LINKED ISSUE #46: Use GitHub Actions to automate docker images pushes
Also: host on the main Stochss-Compute

### PR DESCRIPTION
**Pull request by mdip226**
This pull request will also include 
- [x] updates to default port number
- [x] docker image debug
~~- [ ]  debug self-start dask client?~~

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ def server_start(host, port, debug=True, delegate_config=None):
     if host is None:
         host = "localhost"
     if port is None:
-        port = 1234
+        port = 29681
```
Comment (ethangreen-dev) [inline app.py:10]: This has already been set in `develop`.

File: stochss_compute/api/delegate/dask_delegate.py [hunk 2]
```diff
@@ -52,7 +52,10 @@ def __init__(self, delegate_config: DaskDelegateConfig):
                 print(self.delegate_config.kube_cluster)
 
             else:
+                # [TODO MD 0]
+                print(f"self.client = Client({self.delegate_config.dask_cluster_address}:{self.delegate_config.dask_cluster_port})")
                 self.client = Client(f"{self.delegate_config.dask_cluster_address}:{self.delegate_config.dask_cluster_port}")
+                print(self.client)
```
Comment (ethangreen-dev) [inline stochss_compute/api/delegate/dask_delegate.py:58]: Debug prints need to be removed.

File: stochss_compute/compute_server.py [hunk 3]
```diff
@@ -49,7 +49,8 @@ def post(self, endpoint: Endpoint, sub: str, request: BaseModel = None) -> reque
                 print(f"[{type(request).__name__}] {url}")
                 return requests.post(url, json=request.json())
 
-            except ConnectionError:
+            except ConnectionError as ce:
+                print(ce.strerror)
```
Comment (ethangreen-dev) [inline stochss_compute/compute_server.py:53]: Same here!

**Review by ethangreen-dev — CHANGES_REQUESTED** [review]
Looks good!

**Review by briandrawert — APPROVED** [review]

File: stochss_compute/compute_server.py (hunk 3 above)
Comment (mdip226) [reply stochss_compute/compute_server.py:53]: lol I thought I got those earlier

**Comment by mdip226** [comment]
I think that's it!

**Review by briandrawert — APPROVED** [review]

**Review by ethangreen-dev — APPROVED** [review]
Looks good!

**Pull request merged**